src/hexdoc/_export/resources/assets/hexdoc/lang/en_us.flatten.json5

```diff
@@ -24,6 +24,15 @@
     recipe: {
       show: "Click to show recipes",
       hide: "Click to hide recipes",
+      smelting: {
+        cooktime: "<span class='cooktime-highlight'>{}</span> Seconds",
```

Comment:
the html should be in the jinja, not the lang file

Reply:
it's like that because i made a little macro to handle the below the gui detail lines. Not sure how best to rework it, but it doesn't *really* need that span anyways, just there so I could maybe style it if I wanted

Reply:
where it says `_("hexdoc.recipe.smelting.cooktime").format(recipe.cookingtime / 20))`, you could probably do like, `.format("<span ...>"~(recipe.cookingtime/20)~"</span")`

Reply:
also there probably needs to be a `|safe` somewhere in there, to disable autoescaping the html